net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild

```diff
@@ -1,11 +1,10 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 PYTHON_COMPAT=( python3_{7..9} )
 PYTHON_REQ_USE=( sqlite )
```

Comment:
shouldn't this be a string and not an array? :thinking:

Reply:
You're right, it should be string